sky/jobs/core.py

```diff
@@ -53,7 +54,11 @@ def launch(
     entrypoint = task
     dag_uuid = str(uuid.uuid4().hex[:4])
 
-    dag = dag_utils.convert_entrypoint_to_dag_and_apply_policy(entrypoint)
+    dag = dag_utils.convert_entrypoint_to_dag(entrypoint)
+    # TODO(zhwu): We should only apply policy to dag and save the config file,
+    # instead of having the config file actually being used.
```

Comment:
Not sure I understand this comment.

Reply:
Ahh, this is a remnant. We have fixed this.